README.md

```diff
@@ -1,3 +1,48 @@
 # Akvorado top ASN stats
 
-**WIP** This repository contains some scripts to fetch the top ASN traffic from a clickhouse instance containing [Akvorado](https://github.com/akvorado/akvorado) flow data.
+This repository contains some scripts to fetch the top ASN traffic from a clickhouse instance containing [Akvorado](https://github.com/akvorado/akvorado) flow data.
+
+## Configuration
+
+Inside the file ./akvo-top-asn/akvo-top-asn.conf.ex there is an example config on how to use the collection script.
+
+## Automatic Collection Campaign
+
+Both the Systemd-Timer and the CronJob are configured to collect data every sunday at 00:00. In those examples the script is located inside the 
+`/var/lib` folder please update the locations based on your deployment.
+
+### Systemd
+
+**Systemd One-Shot Unit**
+
+`/etc/systemd/system/dd-ix-export.service`
+```
+[Unit]
+Description=DD-IX Traffic Statistics Collection
+
+[Service]
+Type=oneshot
+ExecStart=/var/lib/akvorado-stats/akvo-top-asn/akvo-top-asn --week 1 --config=/var/lib/akvorado-stats/akvorado-top-asn/akvo-top-asn.conf
```

Comment:
The `--week 1` parameter is already the default. This command won't work as it is missing one of the required action parameters (i.e. `--upload`).